src/topbar/components/KTopbarUI.vue

```diff
@@ -19,8 +18,50 @@
           v-else
           show-loading
         />
-      </RouterLink>
-    </div>
+      </div>
+      <QMenu>
+        <QList>
+          <QItem
+            v-for="group in myGroups"
+            :key="group.id"
+            v-close-popup
+            :to="{ name: 'group', params: { groupId: group.id } }"
+            clickable
+          >
+            <QItemSection
+              avatar
+              class="groups"
+            >
+              <QIcon
+                v-if="!group.hasPhoto"
```

Comment:
Minor: I would swap around the if condition, so have the image first with a `v-if="group.hasPhoto"` and then the users icon as a `v-else` fallback.

Reason: slightly simplifies the if, don't have to think twice, if _not_ ...? and also presents it in the ideal order, the image is the main case, the icon just otherwise...